api.js

```diff
@@ -160,16 +161,17 @@ class Api extends EventEmitter {
 					this._setupTimeout(runStatus);
 				}
 
-				let overwatch;
+				let concurrency = Math.max(2, os.cpus().length || 1);
```

Comment:
> Could you add a comment explaining the logic here? Like, why use 2 on single-core machines?

Will do. It's to make it still concurrent on VMs that report only one CPU: https://github.com/sindresorhus/grunt-concurrent/commit/ef076acedd2bfcac84f9ce038d84af884b12d841 Does that make sense?

> When would os.cpus() return an empty array?

Some years ago someone reported an issue about it returning an empty array for a VM or something. I can't find the issue now, so let's just remove it and see.